### PR TITLE
Added instructor note on layout naming

### LINKED ISSUE #209: Confusing names of layouts
Learners found names of layout templates confusing, especially page.html layout vs. page term used for 'webpage'. Default layout was not much better.

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #209

### CODE REVIEW

**Review by sstevens2 — APPROVED** [review]
This looks great to me Aleks!  Also liked some of the formatting changes for the other sections.

File: _extras/guide.md [hunk 1]
```diff
@@ -2,37 +2,45 @@
 title: "Instructor Notes"
 ---
 
-## Introduction
```
Comment (tobyhodges) [inline _extras/guide.md:5]: I would much prefer to keep the headings for the individual episodes, as I think this makes it much easier for folks to understand which part of the lesson is relevant to each of these notes. Perhaps we could have the episode names as second-level headings and the note titles you have added (which are definitely also helpful!) as third-level, i.e.

```Markdown
## Introduction

### HedgeDoc/CodiMD Collaborative Notes Document

[...]

###  'Add Your Repository Details to HedgeDoc' Exercise 

[...]

## Reusing Blocks of Content

### `includes` vs `include` Confusion

[...]

## Page Layouts

### Layout Name Confusion

[...]
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Thanks @anenadic this is exactly the information we need to include in the Instructor Notes but I would prefer to keep the episode headings in the page as well, to help Instructors better understand what part of the lesson each note is relevant to.

File: _extras/guide.md (hunk 1 above)
Comment (anenadic) [reply _extras/guide.md:5]: Ah, got you @tobyhodges - totally makes sense. I was thinking of improving this PR with links to episodes in the text (I added some of them but not for all notes) but it is much better and clearer to use the episode names as section headers and have all notes for that episode included.

Comment (anenadic) [reply _extras/guide.md:5]: Improved via f926df171568dc98f4bb507d2d6f2c9330d1c92a

File: _extras/guide.md [hunk 2]
```diff
@@ -2,37 +2,59 @@
...
+helping people who are stuck, etc.
+
+## Episode: [Authoring With Markdown](../markdown/index.html)
+
+### 'Add Your Repository Details to HedgeDoc' Exercise
+We recommend doing the ["Optional Exercise: Add Your Repository Details to CodiMD" exercise](../markdown/index.html#optional-exercise-add-your-repository-details-to-codimd) when teaching online, as it is 
+very useful for helpers to have links to all the learner repos **and** rendered websites for troubleshooting. 
+Then when a learner messages in chat, the helper can find their repo and look at the commit history to find the error.
+
+## Episode: [Starting With Jekyll](../starting-jekyll/index.html)
```
Comment (tobyhodges) [inline _extras/guide.md:41]:
```suggestion
## Episode: [Hosting Websites on GitHub](../github-pages/index.html

## Episode: [Starting With Jekyll](../starting-jekyll/index.html)
```

File: _extras/guide.md [hunk 3]
```diff
@@ -2,37 +2,59 @@
...
+
+## Episode: [Page Layouts](../layouts/index.html)
+
+### Layout Names Confusion
+Learners sometimes get confused about conventional names of layouts used in Jekyll - for example `home` 
+layout vs. home page (`index.html`), `page` layout vs. page used as a term for 'webpage', `default` layout was not much better 
+either. While it is not a must, recommendation here is to stick with this conventional terminology as it is 
+reused across different Jekyll websites that learners may come across in future and to spend some time clarifying this. 
+
+## Episode: [Loops and Collections](../filters/index.html)
```
Comment (tobyhodges) [inline _extras/guide.md]:
```suggestion
## Episode: [Working With Filters](../filters/index.html)

## Episode: [Loops and Collections](../arrays/index.html)
```

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**